lib/jsdom/living/nodes/SVGElement.idl

```diff
@@ -0,0 +1,6 @@
+interface SVGElement : Element {
```

Comment:
What to do with the interfaces which are no longer present in the draft you linked (e.g. SVGStylable)?